<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Perform FEA</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="genCountLabel">
       <property name="text">
        <string>There are 0 generations</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="analysedCountLabel">
       <property name="text">
        <string>0 successful analyses</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="startFEAButton">
       <property name="text">
        <string>Start FEA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteAnalyses">
       <property name="text">
        <string>Delete analyses</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Configuration (Advanced)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="growthRateBox">
        <property name="value">
         <double>0.300000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="finenessLabel">
        <property name="text">
         <string>Fineness:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="growthRateLabel">
        <property name="text">
         <string>Growth rate:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="segsPerEdgeLabel">
        <property name="text">
         <string>No. Segments per Edge:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="segsPerRadiusLabel">
        <property name="text">
         <string>No. Segments per Radius:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="segsPerEdgeBox">
        <property name="maximum">
         <number>9999</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="segsPerRadiusBox">
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="finenessBox">
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="currentText">
         <string>UserDefined</string>
        </property>
        <property name="currentIndex">
         <number>5</number>
        </property>
        <item>
         <property name="text">
          <string>VeryCoarse</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Coarse</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Moderate</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>VeryFine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>UserDefined</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
